cli/lib/kontena/cli/apps/logs_command.rb

```diff
@@ -1,72 +1,65 @@
 require_relative 'common'
+require_relative '../helpers/log_helper'
 
 module Kontena::Cli::Apps
   class LogsCommand < Clamp::Command
     include Kontena::Cli::Common
     include Kontena::Cli::GridOptions
+    include Kontena::Cli::Helpers::LogHelper
+
     include Common
 
     option ['-f', '--file'], 'FILE', 'Specify an alternate Kontena compose file', attribute_name: :filename, default: 'kontena.yml'
     option ['-p', '--project-name'], 'NAME', 'Specify an alternate project name (default: directory name)'
-    option ["-l", "--lines"], "LINES", "How many lines to show", default: '100'
+    option ["-l", "--lines"], "LINES", "How many lines to show", default: 100 do |s|
+      Integer(s)
```

Comment:
I used the `Integer(s)` syntax from the clamp examples.

I dropped this change from this PR, it's not really relevant. If I end up refactoring all the CLI logs commands, then I'll fix this for all of them :)